linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/oval/shared.xml

```diff
@@ -10,7 +10,7 @@
       <description>Look for argument fips=1 in the kernel line in /etc/default/grub.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
+      <extend_definition comment="Installed OS is FIPS certified" definition_ref="installed_OS_is_FIPS_certified" />
```

Comment:
I'm a bit confused about what checks we should be evaluating for this rule (and others witch extend definition `installed_OS_is_certified`.

In the rule, description only talks about how to enable FIPS mode, and there is a regulatory warning saying that `...ability to enable FIPS mode does not denote FIPS compliancy or certification.`
So there can be situation 1. **a FIPS certified OS with FIPS mode enabled** and situation 2. **a non-FIPS certified OS with FIPS mode enabled**. 

From rule warning I understand that both situation 1 and 2 should evaluate to pass, because FIPS is enabled.
So this check should actually have `installed_OS_is_certified` removed and not replaced by any other check?

So, as I understand, the verification for FIPS certified system should be done by selecting the rule `intalled_OS_is_FIPS_certified` in relevant profiles.

Reply:
Great question!

I was also confused. I understand the purpose of the `extend_definiton`s to `installed_OS_is_certified` in this kind of rules as a check if installed OS has received FIPS certification. I think that the expectation is that all these rules (eg. fail on a system that isn't FIPS certified.  I don't know if that's correct or not, but it preserves the current behavior.

Reply:
> I think that the expectation is that all these rules (eg. fail on a system that isn't FIPS certified. I don't know if that's correct or not, but it preserves the current behavior.

@jan-cerny you are very correct.

Reply:
So that means the extend_definition is used here correctly and shouldn't be removed. @redhatrises Apart from that, what about @yuumasato 's suggestion to include rule `installed_OS_is_FIPS_certified` in profiles?

Reply:
I gave it some thought, and I consider adding `installed_OS_is_FIPS_certified` rule into profiles could be done if required formally or explicitly. Anyhow, FIPS certification is be checked indirectly by the `_fips_` rules that extend definition `installed_OS_is_FIPS_certified`.